packages/Disclosure/src/disclosure.scss

```diff
@@ -142,6 +142,10 @@
 
 .ids-disclosure .ids-disclosure__header-icon {
     margin: var(--ids-disclosure-header-icon-margin);
+
```

Comment:
This change seems undocumented

Reply:
ya when I was testing I noticed that the variable wasn't written right.. should I add this to my changeset?

Reply:
I updated the changesets